### PR TITLE
Add documentation for dual-write from other

### PR DESCRIPTION
**Pull request by JamesGuthrie**
# Description

This adds documentation for how to do dual-write and backfill from a non-postgresql database to Timescale.

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: _partials/_migrate_dual_write_6a_through_c.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+<Highlight type="important">
+The BETWEEN operator is inclusive of both the start and end ranges, so it is
+not recommended to use it.
+</Highlight>
+
+### 6c. Turn off compression policies in the target for the hypertable
+
+Compression policies must be turned off for the target hypertable while data is
```
Comment (alejandrodnm) [inline _partials/_migrate_dual_write_6a_through_c.md:38]: Should we disable all cagg related refresh policies, and retention policies for the hypertable? 

I'm thinking on the case were we might have wholes in the materialised cagg, due to a refresh triggering when there's no data.

File: migrate/dual-write-and-backfill/dual-write-from-postgres.md.orig [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+FROM _timescaledb_config.bgw_job j
+JOIN _timescaledb_catalog.hypertable h ON h.id = j.hypertable_id
+WHERE j.proc_schema IN ('_timescaledb_internal', '_timescaledb_functions')
+  AND j.proc_name = 'policy_compression'
+  AND j.id >= 1000
+  AND format('%I.%I', h.schema_name, h.table_name)::text::regclass = :'hypertable'::text::regclass;
+EOF
+```
+
+<<<<<<< HEAD
```
Comment (alejandrodnm) [inline migrate/dual-write-and-backfill/dual-write-from-postgres.md.orig:190]: CONFLICT MARKER

**Review by alejandrodnm — CHANGES_REQUESTED** [review]
Haven't reviewed everything. 

The conflict marker needs to go, that's why the request changes.

File: _partials/_migrate_dual_write_6a_through_c.md (hunk 1 above)
Comment (JamesGuthrie) [reply _partials/_migrate_dual_write_6a_through_c.md:38]: Ya, that's a fair point. I kind of made the assumption that at this point people won't have caggs or retention policies configured on their hypertables, but that's probably not a given. I _think_ we should just tell people "don't create caggs on hypertables before you've backfilled, and if you do, ensure that you refresh all caggs over the whole duration when you're done backfilling." WDYT?

Comment (alejandrodnm) [reply _partials/_migrate_dual_write_6a_through_c.md:38]: Right, this is from non-timescale sources, I didn't take that into account.

I agree, we should mention that they shouldn't create those policies until the backfill is ready.

**Review by alejandrodnm — APPROVED** [review]

**Pull request merged**